src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AdministrationController.php

```diff
@@ -149,13 +158,15 @@ protected function processForm(Request $request, FormHandlerInterface $formHandl
 
         if ($form->isSubmitted()) {
             $data = $form->getData();
-            $saveErrors = $formHandler->save($data);
+            try {
+                $formHandler->save($data);
+            } catch (DataProviderException $e) {
+                $this->flashErrors($this->getErrorMessages($e->getDataProviderErrors()));
```

Comment:
Now that you mention it, I guess it would make sense to add default error.

It allows you to throw exception anywhere during the process with some custom error and process will still work.